.changeset/chatty-experts-smell.md

```diff
@@ -0,0 +1,27 @@
+---
+"astro": minor
+---
+
+The CSRF protection feature that was introduced behind a flag in [v4.6.0](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#460) is no longer experimental and is available for general use. If you were using the experimental feature, you'll have to update your `astro.config.mjs` as follows:
```

Comment:
```suggestion
The CSRF protection feature that was introduced behind a flag in [v4.6.0](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#460) is no longer experimental and is available for general use. 

To enable the stable version, add the new top-level `security` option in `astro.config.mjs`. If you were previously using the experimental version of this feature, also delete the experimental flag:
```
This one is a little different from some other cases because they may or may not have been using the experimental flag, but might still want to add it new to the same config now!